std/testing/mock.ts

```diff
@@ -0,0 +1,26 @@
+/** Create dummy Deno.Conn object with given base properties */
+export function mockConn(base: Partial<Deno.Conn> = {}): Deno.Conn {
```

Comment:
Please move this to the http subdir. Maybe we can promote it later, but it seems too specific for std/testing.

Reply:
OK.